build.cake

```diff
@@ -35,11 +36,11 @@ Task("Rebuild")
 
 Task("GenerateDocs")
 	.IsDependentOn("Build")
-	.Does(() => GenerateDocs(verify: false));
+	.Does(() => GenerateDocs(docsAssembly, docsSourceUri, verify: false));
 
 Task("VerifyGenerateDocs")
 	.IsDependentOn("Build")
-	.Does(() => GenerateDocs(verify: true));
+	.Does(() => GenerateDocs(docsAssembly, docsSourceUri, verify: true));
```

Comment:
Why was this necessary? Weird Mono issue?

Reply:
Script-scoped variables can't be referenced from functions when running with the Mono script host.